core/components/molecules/page-header/page-header.md

```diff
@@ -69,3 +69,15 @@ A Page header can also have a single secondary action:
   }
 />
 ```
+
+## Usage
+
+Use the `PageHeader` to add a title, description and main actions to pages.
+
+This component should be placed within the `PageLayout.Header` layout to have the correct spacing between the header of the page and the content.
```

Comment:
Jumped the gun here 😄 

`PageLayout.Header` doesn't exist yet, this will be confusing trying to use PageHeader. Let's comment this out and bring it back when PageLayout is released/stable?